frontend/src/app/testResults/submissions/Submissions.tsx

```diff
@@ -0,0 +1,170 @@
+import { useParams } from "react-router-dom";
+import moment from "moment";
+import { DatePicker, Label } from "@trussworks/react-uswds";
+import { useState } from "react";
+
+import Pagination from "../../commonComponents/Pagination";
+import { formatDateWithTimeOption } from "../../utils/date";
+import {
+  GetUploadSubmissionsQuery,
+  useGetUploadSubmissionsQuery,
+} from "../../../generated/graphql";
+import {
+  setEndTimeForDateRange,
+  setStartTimeForDateRange,
+} from "../../analytics/Analytics";
+import { LoadingCard } from "../../commonComponents/LoadingCard/LoadingCard";
+
+const Submissions = () => {
+  const urlParams = useParams();
+  const pageNumber = Number(urlParams.pageNumber) || 1;
+  const pageSize = 10;
+
+  const [startDate, setStartDate] = useState<Date | null>(null);
+  const [endDate, setEndDate] = useState<Date | null>(null);
+  const [resetCount, setResetCount] = useState<number>(0);
+
+  const { data: submissions, loading, error } = useGetUploadSubmissionsQuery({
+    fetchPolicy: "no-cache",
+    variables: {
+      pageSize,
+      pageNumber: pageNumber - 1,
+      startDate,
+      endDate,
+    },
+  });
+
+  if (loading) {
+    return <LoadingCard />;
+  }
+
+  if (error) {
+    throw error;
+  }
+
+  const SubmissionsTableRows = (
+    submissions: GetUploadSubmissionsQuery | undefined
+  ) => {
+    if (!submissions || submissions.uploadSubmissions.totalElements === 0) {
+      return (
+        <tr>
+          <td>No results</td>
+        </tr>
+      );
+    }
+
+    // `sort` mutates the array, so make a copy
+    return submissions.uploadSubmissions.content.map((submission) => {
+      return (
+        <tr key={submission.internalId}>
+          <td>{submission.reportId}</td>
+          <td>{formatDateWithTimeOption(submission.createdAt, true)}</td>
+          <td>{submission.recordsCount}</td>
+          <td>{submission.status}</td>
```

Comment:
This will always be `PENDING` unless and until we introduce some mechanism to update the status in our own database, right? Do you think we should hold off on a `status` column until we can give accurate data or just roll with it for now?

Reply:
this is a good point, i think we can maybe implement an update status mechanism